packages/kbn-discover-contextual-components/src/data_types/logs/components/summary_column/content.tsx

```diff
@@ -14,7 +14,7 @@ import {
   getLogDocumentOverview,
   getMessageFieldWithFallbacks,
 } from '@kbn/discover-utils';
-import * as constants from '../../../../../common/data_types/logs/constants';
+import * as constants from '@kbn/discover-utils/src/data_types/logs/constants';
```

Comment:
Hmm, I don't think so? Whilst maintaining the semantics of `import * as constants` and allowing potentially other contexts (not just logs) to live in this package.

Reply:
Not sure whether this will cause problems with the sustainable Kibana architecture effort. I guess we'll be told when it does. :shrug: The alternative would be for the `@kbn/discover-utils` package to `export * as logsConstants from '...'`.